incident_viewer_demo/src/App.js

```diff
@@ -77,35 +77,51 @@ var viewMetric = {
 //////////////////////////////////////////////////////////////////////////
 // Done handling the discombobulated variables that the chart takes in. //
 //////////////////////////////////////////////////////////////////////////
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <Row className="Chart-row">
-          <AutoWidth>
-            <LineChartWithCounts
-              id={chartId}
-              colors={colors}
-              counts={counts}
-              highlightCounts={clientIspCounts}
-              series={clientIspTimeSeriesData}
-              annotationSeries={annotationTimeSeries}
-              onHighlightDate={onHighlightTimeSeriesDate}
-              highlightDate={highlightTimeSeriesDate}
-              onHighlightLine={onHighlightTimeSeriesLine}
-              highlightLine={highlightTimeSeriesLine}
-              yFormatter={viewMetric.formatter}
-              xKey="date"
-              yAxisLabel={viewMetric.label}
-              yAxisUnit={viewMetric.unit}
-              yKey={viewMetric["datakey"]}
-            />
-          </AutoWidth>
-        </Row>
-      </header>
-    </div>
-  );
-}
+class App extends React.Component {
```

Comment:
I can't comment on the actual line, but please add a comment indicating that the variables myMoment and counts above are for hard coding the background data of the graph. It is also not clear what the clientIspCounts are so please add a comment there as well.